Mathlib/Geometry/Manifold/Instances/Real.lean

```diff
@@ -325,7 +329,7 @@ instance IccManifold (x y : ℝ) [h : Fact (x < y)] :
 
 /-- The manifold structure on `[x, y]` is smooth.
 -/
-instance Icc_smooth_manifold (x y : ℝ) [Fact (x < y)] :
```

Comment:
Similarly, I'd prefer to think again: what's the auto-generated name? (The current name is not good, renaming to IccSmoothManifoldWithCorners is fine with me.)

Reply:
`Icc_smoothManifoldWithCorners` because `SmoothManifoldWithCorners` is a `Prop`. The instance below that could be renamed `Icc_zero_one_smoothManifoldWithCorners`?

Reply:
Agreed on the Prop. I think omitting `zero_one` is clearer.